src/item.rs

```diff
@@ -307,13 +307,12 @@ impl Printer {
         self.word("type ");
         self.ident(&item.ident);
         self.generics(&item.generics);
-        self.where_clause_oneline(&item.generics.where_clause);
-        self.word("= ");
+        self.word(" = ");
         self.neverbreak();
         self.ibox(-INDENT);
         self.ty(&item.ty);
         self.end();
-        self.word(";");
+        self.where_clause_oneline_semi(&item.generics.where_clause);
```

Comment:
I don't think this change is correct. Where-clause is not allowed after the type on type aliases.

https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=19a4e998272aede03ba0d8aae2938620

Reply:
Wow true. Reverted for here.